src/routes/contact.js

```diff
@@ -1,10 +1,46 @@
 const express = require("express");
+const nodemailer = require("nodemailer");
+const bodyParser = require("body-parser");
 const policy = require("../../data/policy.json");
 
+const app = express();
 const router = express.Router();
 
-router.get("/contact", (req, res) => {
-  res.render("contact", policy);
-});
+app.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
Maybe we can do it in `src/index.js`?
This here looks like a hack 🙂

Reply:
Just realized we already called it in `src/index.js`. Will remove it 🙂